dbms/src/AggregateFunctions/AggregateFunctionNull.h

```diff
@@ -629,6 +629,21 @@ class AggregateFunctionNullUnaryForWindow
         }
     }
 
+    void batchInsertSameResultInto(ConstAggregateDataPtr __restrict place, IColumn & to, size_t num, Arena *)
+        const override
+    {
+        auto & to_concrete = static_cast<ColumnNullable &>(to);
+        if (getCounter(place) > 0)
+        {
+            nested_function->batchInsertSameResultInto(nestedPlace(place), to_concrete.getNestedColumn(), num, nullptr);
+            to_concrete.insertManyNulls(num, 0);
```

Comment:
change to `insertManyNonNulls`?

Reply:
> change to `insertManyNonNulls`?

I think that we do not have this function yet.

Reply:
This `insertManyNulls` looks very confusing to me, why not just use `to_concrete.getNullMapData().resize_fill()`?

Reply:
> This `insertManyNulls` looks very confusing to me, why not just use `to_concrete.getNullMapData().resize_fill()`?

done